terraform/pilot/main.tf

```diff
@@ -15,15 +15,20 @@ variable "branch" {}
 
 variable "security_context" { default = "dev" }
 
-variable "aws_region" { default = "us-east-1" }
-variable "ami_type" { default = "hardened" }
+variable "network" { 
+    type = "map"
+    default = {
+        region = "us-east-1" 
+    }
+}
+variable "ami_type" { default = "not-hardened" }
 variable "bastion_subnet_id" {}
 variable "bastion_security_group_id" {}
-variable "bastion_instance_type" {}
+variable "bastion_instance_type" { default = "m3.medium" }
```

Comment:
Yep that's the formula Dan and I have been using for many of the app servers. But what if it's not anything-heavy as would likely be the case for bastion server? t2.medium?